test/sql/pg_catalog/sqlalchemy.test

```diff
@@ -4,6 +4,9 @@
 
 # https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/dialects/postgresql/base.py
 
+statement ok
+CREATE TYPE greeting AS ENUM('hi', 'bonjour', 'konnichiwa', 'howdy')
+
```

Comment:
There's another query on the bottom that concerns enums (under `# load enums`):

```sql
SELECT t.typname as "name",
	-- no enum defaults in 8.4 at least
	-- t.typdefault as "default",
	pg_catalog.pg_type_is_visible(t.oid) as "visible",
	n.nspname as "schema",
	e.enumlabel as "label"
FROM pg_catalog.pg_type t
		LEFT JOIN pg_catalog.pg_namespace n ON n.oid = t.typnamespace
		LEFT JOIN pg_catalog.pg_enum e ON t.oid = e.enumtypid
WHERE t.typtype = 'e'
```

Does that now also work/produce the same result as Postgres?

Reply:
I will look into it

Reply:
I've updated the pr and this test now gives the same result as postgres.